core/execution/common/market_activity_tracker.go

```diff
@@ -434,7 +433,7 @@ func (mat *MarketActivityTracker) UpdateFeesFromTransfers(asset, market string,
 			mat.addFees(mt.makerFeesPaid, t.Owner, t.Amount.Amount, mt.totalMakerFeesPaid)
 		case types.TransferTypeMakerFeeReceive:
 			mat.addFees(mt.makerFeesReceived, t.Owner, t.Amount.Amount, mt.totalMakerFeesReceived)
-		case types.TransferTypeLiquidityFeeDistribute:
+		case types.TransferTypeLiquidityFeeNetDistribute:
 			mat.addFees(mt.lpFees, t.Owner, t.Amount.Amount, mt.totalLpFees)
```

Comment:
@ze97286 could we include `TransferTypeSlaPerformanceBonusDistribute` as well here?

Reply:
I mean, we could, but should we?

Reply:
Also, if you are the only LP in a market and you only just meet the required time on book fraction, all your LP rewards will come to you in a `TransferTypeSlaPerformanceBonusDistribute` transfer and I would expect the LP to still be rewarded.

Reply:
I mean, not strictly, it's a metric for now much fees LP is getting, so why include bonuses? it's not like we're including penalties the other way around. Anyways, if you want it you got it.

Reply:
Yeah but the final fees the LP receives is NetDistribute + BonusDistribute.

Reply:
Nice thank you 🙏 

Market-sim plots are looking good now, will approve.